src/components/LanguageSwitch/LanguageSwitch.tsx

```diff
@@ -102,10 +106,8 @@ const LanguageSwitch = () => {
     ),
     [getCurrentLngDetails]
   );
-
-  const optionLabel = useCallback(({ translation }: ReturnType<typeof getCurrentLngDetails>) => translation, [
-    getCurrentLngDetails,
-  ]);
+  // prettier-ignore
```

Comment:
Please why did you add 'prettier-ignore' here?

Reply:
I can not pass lint check because of this line, but I don't know how to fix it.
So I add prettier-ignore to try but I forget removing this.
Sorry for confused you